frontend/public/locales/en/renew-adult-child.json

```diff
@@ -0,0 +1,14 @@
+{
+  "marital-status": {
+    "page-title": "Marital status",
+    "marital-status": "What is your current marital status?",
+    "back-btn": "Back",
+    "continue-btn": "Continue",
+    "cancel-btn": "Cancel",
+    "save-btn": "Save",
+    "error-message": {
+      "marital-status-required": "Select marital status",
+      "marital-status-no-partner-information": "If you are married or in a common-law relationship you must enter their information on the next page. Select your marital status and press 'Save'."
```

Comment:
FYI the Figma has the form for the partner information on the same page as the marital status radio buttons.

Reply:
yeah, so to me it's really ambiguous what the designers want in this case (I  wasn't invited to a design session).  Do you know what the expected flow is like?

Reply:
My understanding: if user selects "Married" or "Common-Law", render the Partner Information section on the same Marital Status page (unlike the benefit application flow).

I also understand you love that idea.